docs_src/utilities/Color.svelte

```diff
@@ -0,0 +1,76 @@
+<script>
+  import Prism from 'svelte-prism';
+  import { Card } from '../../src';
+
+  function moveToAllColors() {
+    document.getElementById('all-colors-section').scrollIntoView();
+  }
+</script>
+
+<h1 class="color--eb">Color</h1>
+
+<h3>Text color</h3>
+<p>color-- sætter farven på elementet</p>
+<p class="color--red" on:click={moveToAllColors} style="cursor: pointer;">Se alle farvemuligheder her</p>
+
+<Card className="bg--graa7">
+  <div class="flex flex-justify--between flex-wrap--wrap" slot="content">
+    <div class="flex flex-align--center padding-xl width-1of3 margin-l bg--white color--black">color--black</div>
+    <div class="flex flex-align--center padding-xl width-1of3 margin-l bg--black color--white">color--white</div>
+    <div class="flex flex-align--center padding-xl width-1of3 margin-l bg--white color--sport">color--sport</div>
+    <div class="flex flex-align--center padding-xl width-1of3 margin-l bg--white color--eb">color--eb</div>
+    <div class="flex flex-align--center padding-xl width-1of3 margin-l bg--white color--orangedark">
+      color--orangedark
+    </div>
+    <div class="flex flex-align--center padding-xl width-1of3 margin-l bg--white color--nyheder">color--nyheder</div>
+    <div class="flex flex-align--center padding-xl width-1of3 margin-l bg--white color--section">
+      color--section <br />(arver sektionsfarven f.eks sport)
```

Comment:
er mere til `color--section <em>(arver sektionsfarven f.eks sport)</em>`
som du nok ved er jeg ikke meget for `<br>` tags